users.json

```diff
@@ -34,4 +34,9 @@
         "link": "https://tanut2539.github.io/portfolio",
         "image": "https://i.ibb.co/RTh56dk/tanut.png"
     }
-]
\ No newline at end of file
+        {
+        "name": "Krati Agarwal",
+        "link": "https://agarkra.github.io/kratiagarwal.github.io/",
+        "image": "https://ibb.co/tp8SDjv"
+    }
```

Comment:
This should be a raw URL pointing to an image of your website.

Reply:
@agarkra please use this image link instead: https://i.ibb.co/RP1fVV1/krati.png